Outreachy Dec 2021 to Mar 2022.md

```diff
@@ -0,0 +1,36 @@
+* We recommend that you conduct learner feedback sessions on at least 3 consecutive chapters (either Web/Android Platform) on any of the math topics inside [Oppia Classroom](https://docs.google.com/document/d/1j1S9HnoesGZMNeYccV8rXZ2bZwPhKV5-B6j9CnPRM_I/edit?usp=sharing). One student is enough for the contributing period, but you can work with more if you want to.
+*  Remember that for the sessions, you will have to make sure that the chapters you will use and the materials (surveys and tests) listed in the guide/checklist are translated in the student's language. Also remember that the report on your findings is your main deliverable for this task, and we recommend you follow the report template listed on the guide/checklist to save effort and don't miss relevant information. 
+
+### Reaching and defining pilots plan with NGOs
+
+* For reaching and drafting pilot plans with NGOs, we recommend that you start finding and listing a few NGOs related to children education that you could potentially work with to use Oppia lessons. You will need to contact and meet with the NGOs in order to introduce Oppia and talk about the NGO activities. 
+* In this meeting, try to understand their pains and needs to better find a way for us to work together in a mutual beneficial partnership. You may make a copy and personalize this [presentation slides ](https://docs.google.com/presentation/d/1kWXni2omdpOZjUId712odYGSxfEziHg0wu81ChdvHAE/edit?usp=sharing) to help you pitch Oppia to the partner.
+* After you have a good understanding of the NGO context, work with them to draft a pilot plan detailing how they can include Oppia in their activities. If you think you need help on how to prioritize which NGOs to reach and how elaborate a nice reach out message, you can email our mentors for more materials.
+
+### Reaching and defining pilots plan with NGOs
+* For this task, you will need to create a plan on how you would scale the lessons and provide them to a larger audience in your country. It's important for the plan to be supported by reliable research that can include: (a) a benchmark analysis of similar products in your region, (b) an analysis of your country's/region's demographics, and (c) an analysis of existing NGOs and potential partners. 
+* You can use any documents and research you find relevant to support your proposal. The format of the plan is completely up to you. 
+* If you need help with the background research for your plan, you can contact the mentors for further guidance and additional materials.
+
+## FAQ
+
+**Q: How can I contribute with the project?**
+
+A: After finishing the required tasks, add a contribution URL to your Outreachy portal with all the necessary details and links. (The Contribution URL for each task would be a link to a Google Drive where you have uploaded the document/spreadsheet for that specific task.) Make sure that the file access is public!
```

Comment:
```suggestion
A: After finishing the required tasks, add a contribution URL to your Outreachy portal with all the necessary details and links. The contribution URL for each task should be a link to a Google Drive folder where you have uploaded the document/spreadsheet for that specific task. Make sure that the file access is public!
```

The part about Google Drive seems pretty important, so here are suggestions for making it stronger